src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -2160,6 +2163,55 @@ def test_aks_azure_service_mesh_enable_disable(self, resource_group, resource_gr
             self.is_empty(),
         ])
 
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2')
+    def test_aks_machine_cmds(self, resource_group, resource_group_location):
```

Comment:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=103693531&view=results) to validate the change.

Please commit the generated recording file if the test passes

- test_aks_machine_cmds